homeassistant/components/azure_event_hub/__init__.py

```diff
@@ -1,88 +1,246 @@
 """Support for Azure Event Hubs."""
+import asyncio
 import json
 import logging
+import queue
+import threading
+import time
 from typing import Any, Dict
 
-from azure.eventhub import EventData, EventHubClientAsync
+from azure.eventhub import (
+    EventData,
+    EventDataBatch,
+    EventHubProducerClient,
+    EventHubSharedKeyCredential,
+)
+from azure.eventhub.exceptions import EventHubError
 import voluptuous as vol
 
+from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import (
     EVENT_HOMEASSISTANT_STOP,
-    EVENT_STATE_CHANGED,
+    MATCH_ALL,
     STATE_UNAVAILABLE,
     STATE_UNKNOWN,
 )
-from homeassistant.core import Event, HomeAssistant
+from homeassistant.core import Event, HomeAssistant, callback
 import homeassistant.helpers.config_validation as cv
-from homeassistant.helpers.entityfilter import FILTER_SCHEMA
```

Comment:
Our core filter logic is required if you want provide a filtering like you do. An past everyone had his own and we streamline this to one logic over all components

Reply:
made this change, but this does not appear to be marked as resolved...